component/src/test/java/org/wso2/extension/siddhi/store/influxdb/ContainsInInfluxDBTestCase.java

```diff
@@ -0,0 +1,250 @@
+
+    @BeforeClass
+    public static void startTest() {
+
+        log.info("==InfluxDB Table contains tests started ==");
+    }
+
+    @AfterClass
+    public static void shutdown() {
+
+        log.info("== InfluxDB Table contains tests completed ==");
+    }
+
+    @BeforeMethod
+    public void init() {
+
+        try {
+            InfluxDBTestUtils.initDatabaseTable(TABLE_NAME);
+        } catch (InfluxDBException e) {
+            log.info("Test case inored due to :" + e.getMessage());
+        }
+    }
+
+    @Test(description = "Test contains with one condition.")
+    public void containsCheckTestWithSingleCondition() throws InterruptedException, InfluxDBException {
+
+        log.info("containsCheckTestWithSingleCondition");
+        SiddhiManager siddhiManager = new SiddhiManager();
+        String streams = "" +
+                "define stream FooStream (name string,value long);\n" +
+                "define stream StockStream (symbol string, price float, volume long,time long);\n" +
+                "@Store(type=\"influxdb\", url = \"" + URL + "\" ," +
+                "username=\"" + USERNAME + "\", password=\"" + PASSWORD + "\", database = \"" + DATABASE
+                + "\")\n" +
+                "@Index(\"symbol\",\"time\")" +
+                "define table StockTable (symbol string, price float, volume long,time long);\n";
+
+        String query = "" +
+                "@info(name = 'query1')\n" +
+                "from StockStream\n" +
+                "select *\n" +
+                "insert into StockTable;\n" +
+                "@info(name = 'query2')\n " +
+                "from FooStream \n" +
+                "[(StockTable.symbol ==name ) in StockTable]\n" +
+                "insert into OutputStream;";
+
+        SiddhiAppRuntime siddhiAppRuntime = siddhiManager.createSiddhiAppRuntime(streams + query);
+        InputHandler stockStream = siddhiAppRuntime.getInputHandler("StockStream");
+        InputHandler fooStream = siddhiAppRuntime.getInputHandler("FooStream");
+
+        siddhiAppRuntime.addCallback("query2", new QueryCallback() {
+            @Override
+            public void receive(long l, Event[] events, Event[] events1) {
+
+                EventPrinter.print(l, events, events1);
+                if (events != null) {
+                    eventArrived = true;
+                } else {
+                    eventArrived = false;
+                }
+            }
+        });
+
+        siddhiAppRuntime.start();
+
+        stockStream.send(new Object[]{"WSO2", 55.6f, 100L, 1548181800007L});
+        stockStream.send(new Object[]{"IBM", 65.6f, 10L, 1548181800500L});
+        stockStream.send(new Object[]{"CSC", 65.6f, 10L, 1548181800800L});
+
+        fooStream.send(new Object[]{"WSO2", 100});
+        fooStream.send(new Object[]{"IBM", 10});
+        fooStream.send(new Object[]{"WSO22", 100});
+
+        SiddhiTestHelper.waitForEvents(waitTime, 2, eventCount, timeout);
+
+        Thread.sleep(1000);
```

Comment:
Why the thread.sleep is used? Since waitForEvents function is called?

Reply:
https://github.com/wso2-extensions/siddhi-store-influxdb/pull/1/commits/520b093086a08e2d8d9df4297dcd71914e15813b